Compiling.md

```diff
@@ -478,6 +478,77 @@ Another method of compiling is using the project [Building for Android with Dock
 
 Compilation of dependent libraries, **leptonica** and **tiff**, are included and handled as well.
 
+## Cross compile on Linux with NDK
+Another method of compiling is doing it on Linux machine, with Android NDK r22 (22.1.7171670). This method compiles for following versions and architectures:
+
+
+| Arch \ Version | 4.1.0 |
+|----------------|---------|
+| armv7-a | ✔ |
+| arm64-v8a | ✔ |
+| x86 | ✔ |
+| x86_64 | ✔ |
+
+
+These prerequisites will be needed:
+* libjpeg - GitHub branch 2.1.1 - https://github.com/libjpeg-turbo/libjpeg-turbo
+* libpng - GitHub branch v1.6.37 - https://github.com/glennrp/libpng
+* libtiff - version 4.0.10 downloaded - https://download.osgeo.org/libtiff/
+* leptonica - version 1.74.4 downloaded - https://github.com/DanBloomberg/leptonica
```

Comment:
Would it also work with the latest Leptonica release 1.81.1?

Reply:
Not sure, that is untested. Let's leave it like this for now.